src/configuration/services/oracle-mysql.md

```diff
@@ -0,0 +1,226 @@
+# Oracle MySQL (Database service)
+
+MySQL is a relational database system.
+
+See the [MySQL documentation](https://docs.oracle.com/cd/E17952_01/index.html) for more information.
+
+## Supported versions
+
+* 5.7
+* 8.0
+
+## Relationship
+
+The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](/development/variables.md#platformsh-provided-variables):
+
+{% codesnippet "https://examples.docs.platform.sh/relationships/oracle-mysql", language="json" %}{% endcodesnippet %}
+
+## Usage example
+
+In your `.platform/services.yaml`:
+
+```yaml
+mydatabase:
+    type: oracle-mysql:8.0
+    disk: 1024
+```
+
+Note that the minimum disk size for mysql is 256MB.
+
+In your `.platform.app.yaml`:
+
+```yaml
+relationships:
+    database: "mydatabase:mysql"
+```
+
+You can then use the service in a configuration file of your application with something like:
+
+{% codetabs name="PHP", type="php", url="https://examples.docs.platform.sh/php/oraclemysql" -%}
+
+{%- language name="Node.js", type="js", url="https://examples.docs.platform.sh/nodejs/oraclemysql" -%}
+
+{%- language name="Python", type="py", url="https://examples.docs.platform.sh/python/oraclemysql" -%}
+
+{%- language name="Go", type="go" -%}
+// Using the Platform.sh Go helper library: https://github.com/platformsh/gohelper
+
+dbString, err := pi.SqlDsn("database")
+if (err != nil) {
+  panic(err)
+}
+
+db, err := sql.Open("mysql", dbString)
+if (err != nil) {
+  panic(err)
+}
+{%- endcodetabs %}
+
+> **note**
+> MySQL schema names can not use system reserved namespace. (mysql, information_schema, etc)
```

Comment:
This note is accurate, but seems out of place here.  Custom schema names aren't mentioned until the next section, so this note should go there instead.